### PR TITLE
Set 'project' attribute when importing 'google_logging_project_sink'

### PR DESCRIPTION
**Pull request by ewbankkit**
Fixes https://github.com/terraform-providers/terraform-provider-google/issues/1018.

Incorporated the import acceptance test into the main acceptance tests as per current best practice.

Modified acceptance test **before** setting `project` attribute:
```
make testacc TEST=./google TESTARGS='-run=TestAccLoggingProjectSink_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./google -v -run=TestAccLoggingProjectSink_basic -timeout 120m
=== RUN   TestAccLoggingProjectSink_basic
--- FAIL: TestAccLoggingProjectSink_basic (5.94s)
	testing.go:513: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
		
		(map[string]string) {
		}
		
		
		(map[string]string) (len=1) {
		 (string) (len=7) "project": (string) (len=11) "root-999999"
		}

```

and **after** setting `project` attribute:
```
make testacc TEST=./google TESTARGS='-run=TestAccLoggingProjectSink_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./google -v -run=TestAccLoggingProjectSink_basic -timeout 120m
=== RUN   TestAccLoggingProjectSink_basic
--- PASS: TestAccLoggingProjectSink_basic (5.56s)
PASS
ok  	github.com/terraform-providers/terraform-provider-google/google	11.741s
```

### CODE REVIEW

**Comment by ewbankkit** [comment]
`Computed: true` is now set for the `project` attribute - I think this breaks backwards compatibility. Shall I write a state migration function?

File: google/resource_logging_project_sink.go [hunk 1]
```diff
@@ -96,3 +104,15 @@ func resourceLoggingProjectSinkDelete(d *schema.ResourceData, meta interface{})
 	d.SetId("")
 	return nil
 }
+
+func resourceLoggingProjectSinkImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	parts := strings.Split(d.Id(), "/")
+	if len(parts) != 4 {
```
Comment (emilymye) [inline google/resource_logging_project_sink.go]: Can you add a regex check and a default project check here? Similar to [KMS key ring](https://github.com/terraform-providers/terraform-provider-google/blob/cc5f77e32e927c3df81fab6abc4ee791c39bab5c/google/resource_kms_key_ring.go#L133-L160)

File: google/import_logging_project_sink_test.go [hunk 2]
```diff
@@ -1,31 +0,0 @@
-package google
```
Comment (emilymye) [inline google/import_logging_project_sink_test.go]: I know it's slightly redundant but I'd prefer not removing this test file and just adding the import tests here.

Comment (ewbankkit) [reply google/import_logging_project_sink_test.go]: Done.

File: google/resource_logging_project_sink.go (hunk 1 above)
Comment (ewbankkit) [reply google/resource_logging_project_sink.go]: Done. Called `parseLoggingSinkId`.

**Comment by emilymye** [comment]
Thanks!

make testacc TEST=./google TESTARGS='-run=TestAccLoggingProjectSink_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./google -v -run=TestAccLoggingProjectSink_ -timeout 120m
=== RUN   TestAccLoggingProjectSink_importBasic
=== RUN   TestAccLoggingProjectSink_basic
=== RUN   TestAccLoggingProjectSink_uniqueWriter
=== RUN   TestAccLoggingProjectSink_updatePreservesUniqueWriter
--- PASS: TestAccLoggingProjectSink_importBasic (2.95s)
--- PASS: TestAccLoggingProjectSink_basic (3.21s)
--- PASS: TestAccLoggingProjectSink_uniqueWriter (4.03s)
--- PASS: TestAccLoggingProjectSink_updatePreservesUniqueWriter (6.52s)
PASS
ok  	github.com/terraform-providers/terraform-provider-google/google6.700s

**Pull request merged**